runing/services.py

```
@@ -0,0 +1,41 @@
+        self.user_repository = user_repository
+
+    def create_user(self, user: schemas.UserCreate) -> schemas.User:
+
+        db_user = self.user_repository.get_user_by_email(user.email)
+        if db_user:
+            raise HTTPException(status_code=400, detail="Email already registered")
+
+        created_user = self.user_repository.create_user(user)
+        return schemas.User.model_validate(created_user)
+
+    def get_user(self, user_id: int) -> schemas.User:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)